libs/utils/Cargo.toml

```diff
@@ -10,6 +10,10 @@ default = []
 # which adds some runtime cost to run tests on outage conditions
 testing = ["fail/failpoints"]
 
+# Enables debugging functionality that's based on the `tracing` crate,
+# e.g., tokio-console or tracing-chrome.
+tracing-based-debugging = [ "console-subscriber", "tracing-chrome", "tracing-flame", "tokio/tracing" ]
+
```

Comment:
Only if the `tracing-based-debugging` feature is enabled shall we spend time compiling these crates / the tokio `tracing` feature code.